api/app/Models/PersonalExperience.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+use Illuminate\Database\Eloquent\Relations\HasMany;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Contracts\Auth\Authenticatable;
+use Illuminate\Auth\Authenticatable as AuthenticatableTrait;
+
+/**
+ * Class PersonalExperience
+ *
+ * @property int $id
+ * @property string $title
+ * @property string $description
+ * @property Illuminate\Support\Carbon $start_date
+ * @property Illuminate\Support\Carbon $end_date
+ * @property string $details
+ * @property Illuminate\Support\Carbon $created_at
+ * @property Illuminate\Support\Carbon $updated_at
+ */
```

Comment:
Do we need a field for *I am comfortable sharing this information publicly* or is that just required to submit the form?

Reply:
I feel like that's probably just required to submit the form. If they don't consent we probably shouldn't be saving it at all.

Reply:
@Jerryescandon ?

Reply:
But let me verify that there's no obscure legal loophole that requires us to "save" this as some form of consent.

Reply:
Man, what a can of worms this was. In short terms yeah. This would be a "required" field. But we are updating the text to be more clear on the intent.

"I agree to share this information with verified Government of Canada hiring managers and HR advisors who have access to this platform."